src/nbl/asset/utils/IShaderCompiler.cpp

```diff
@@ -110,27 +111,17 @@ namespace nbl::asset::impl
         {
             shaderc_include_result* res = new shaderc_include_result;
             std::string res_str;
-            std::filesystem::path relDir;
-            const bool reqFromBuiltin = asset::IIncludeHandler::isBuiltinPath(_requesting_source);
-            const bool reqBuiltin = asset::IIncludeHandler::isBuiltinPath(_requested_source);
-            if (!reqFromBuiltin && !reqBuiltin)
-            {
-                //While #includ'ing a builtin, one must specify its full path (starting with "nbl/builtin" or "/nbl/builtin").
-                //  This rule applies also while a builtin is #includ`ing another builtin.
-                //While including a filesystem file it must be either absolute path (or relative to any search dir added to asset::iIncludeHandler; <>-type),
-                //  or path relative to executable's working directory (""-type).
-                relDir = std::filesystem::path(_requesting_source).parent_path();
-            }
-
```

Comment:
this needs to stay, it basically prevents you from breaking out of `nbl/builtin` once you're inside it (makes sure our "system headers" can't include anything from the outside, which saves us a bunch of bugs)